assets/dev/scss/frontend/_global.scss

```diff
@@ -26,9 +26,6 @@
 	img {
 		height: auto;
 		max-width: 100%;
-		border: none;
-		border-radius: 0;
-		box-shadow: none;
```

Comment:
And here?

Reply:
These rules are now controlled by the Theme Style editor.